omnigibson/prims/rigid_prim.py

```diff
@@ -211,7 +212,7 @@ def update_meshes(self):
 
                     volume, com = get_mesh_volume_and_com(mesh_prim)
                     # We need to transform the volume and CoM from the mesh's local frame to the link's local frame
-                    local_pos, local_orn = mesh.get_local_pose()
+                    local_pos, local_orn = mesh.get_position_orientation(frame="parent")
```

Comment:
sometimes we pass in `frame="parent"`, sometimes we pass in `"parent"`. can we always pass in `frame="parent"` or `frame="scene"` to be consistent?